<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_targetList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>TargetList</string>
    </property>
    <widget class="QListWidget" name="listWidget_ips">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>localhost</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>192.168.0.1/24</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_addip">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delip">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cleanip">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_portList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PortList</string>
    </property>
    <widget class="QListWidget" name="listWidget_ports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>80</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>53</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8080</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8090</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_addport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cleanport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_scan">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>381</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>761</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Service</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Banner</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>localhost:8080</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Apache2.7</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_settings">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Speed: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Type: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>synScan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tcpScan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>udpScan</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>kbps</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_speed">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>synScan</string>
     </property>
     <item>
      <property name="text">
       <string>synScan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tcpScan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>udpScan</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
